packages/components/organisms/f-checkout/src/store/tests/checkoutAnalytics.module.test.js

```diff
@@ -402,6 +402,64 @@ describe('CheckoutAnalyticsModule', () => {
                 expect(window.dataLayer).toContainEqual(expected);
             });
         });
+
+        describe('trackLowValueOrderExperiment ::', () => {
+            it('should `push` low value order event to data layer if it is returned in request header`', () => {
+                // Arrange
+                const expected = {
+                    custom: {
+                        experiment: {
+                            id: 'EX-1862',
+                            name: 'low_value_order_threshold',
+                            platform: 'experiment_api',
+                            variant: {
+                                name: 'test'
+                            },
+                            version: 1
+                        }
+                    },
+                    event: 'trackExperimentV2'
+                };
+
+                const mockedResponseHeaders = {
+                    [HEADER_LOW_VALUE_ORDER_EXPERIMENT]: 'test'
+                };
+
+                // Act
+                trackLowValueOrderExperiment(mockedResponseHeaders);
+
+                // Assert
+                expect(window.dataLayer).toContainEqual(expected);
+            });
+
+            it('should not `push` low value order event to data layer if it is not returned in request header`', () => {
```

Comment:
Minor: The backtick at the end of this IT statement should be a normal single quote right?